### ISSUE TITLE
proper headless mode

### ISSUE THREAD

**Issue by eps696** [comfyanonymous/ComfyUI#2394]
is it planned to develop ComfyAPI up to the point when any workflow can be loaded and run from the command line without extra coding (like manual assignments of the nodes in the python script)?

**Comment by ltdrdata** [comment]
Perhaps this is what you're looking for.

https://github.com/pydn/ComfyUI-to-Python-Extension

**Comment by eps696** [comment]
not really; i expect some way to run a command (used in batches) triggering comfy to perform the actions, rather than externally reproducing the pipeline. this may serve as an interim solution though, thanks for the link.

**Comment by realazthat** [comment]
Slightly related: You can use the API (https://github.com/comfyanonymous/ComfyUI/tree/master/script_examples).

But anything can break with future developments.

**Comment by shiimizu** [comment]
> [Apapach](https://codeberg.org/tekakutli/apapach): An effort to use Comfy as a Diffusers alternative (python control of the graph), and an Emacs-UI built on top of that. <sup> [1](https://github.com/WASasquatch/comfyui-plugins)</sup>

**Comment by mizukarada** [comment]
> [Apapach](https://codeberg.org/tekakutli/apapach): An effort to use Comfy as a Diffusers alternative (python control of the graph), and an Emacs-UI built on top of that. <sup> [1](https://github.com/WASasquatch/comfyui-plugins)</sup>
